Source/Krypton Components/Krypton.Ribbon/Controller/AppButtonController.cs

```diff
@@ -250,10 +251,15 @@ public virtual void LostFocus([DisallowNull] Control c)
         /// <param name="e">A KeyEventArgs that contains the event data.</param>
         public void KeyDown(Control c, KeyEventArgs e)
         {
-            ViewBase newView = null;
+            ViewBase? newView = null;
             var ribbon = (KryptonRibbon)c;
 
-            switch (e.KeyData)
+            if ( ribbon.TabsArea is null )
```

Comment:
Because `ribbon` may be null now (Due to `as`) then also check it in this `if` check please ?

Apply these changes to the other locations as well